src/lightning_app/core/flow.py

```diff
@@ -276,7 +276,11 @@ def state_with_changes(self):
     @property
     def flows(self):
         """Return its children LightningFlow."""
-        return {el: getattr(self, el) for el in sorted(self._flows)}
+        flows = {el: getattr(self, el) for el in sorted(self._flows)}
+        for struct_name in sorted(self._structures):
+            for flow in getattr(self, struct_name).flows:
+                flows[flow.name] = flow
+        return flows
```

Comment:
This is a breaking change, everywhere the `.flows` is used it expects to get only direct children not to get nested flows in structures. maybe we can have it in a different method/property?

Reply:
Although this is probably an unintended effect of the previous code. Semantically it was more a bug than a feature IMO.

Reply:
Yes. It was a bug I found while developing this feature.